caravel/migrations/versions/ad82a75afd82_add_query_model.py

```diff
@@ -35,15 +35,23 @@ def upgrade():
         sa.Column('rows', sa.Integer(), nullable=True),
         sa.Column('error_message', sa.Text(), nullable=True),
         sa.Column('start_time', sa.DateTime(), nullable=True),
+        sa.Column('changed_on', sa.DateTime(), nullable=True),
         sa.Column('end_time', sa.DateTime(), nullable=True),
         sa.ForeignKeyConstraint(['database_id'], [u'dbs.id'], ),
         sa.ForeignKeyConstraint(['user_id'], [u'ab_user.id'], ),
         sa.PrimaryKeyConstraint('id')
     )
     op.add_column('dbs', sa.Column('select_as_create_table_as', sa.Boolean(),
                                    nullable=True))
+    op.create_index(op.f('ix_query_changed_on'), 'query', ['changed_on'],
```

Comment:
We need a composite index here, not 2 indexes. Here's an example:
https://github.com/apache/incubator-airflow/blob/master/airflow/models.py#L708